### PR TITLE
[sleeplog] Improved README + option to add functions triggered by sleeplog

### PR DESCRIPTION
**Pull request by storm64**
- Improve README.md.
- Add the option to insert functions from other apps, that are triggered by status changes and/or within a specified time period.
- Remove old log conversion function for log files before 0.10, as it has some bugs.

### CODE REVIEW

File: apps/sleeplog/README.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
-This app logs and displays the following states:  
+This app logs and displays the following states:
 - sleepling status: _unknown, not worn, awake, light sleep, deep sleep_
 - consecutive sleep status: _unknown, not consecutive, consecutive_
 
 It is using the built in movement calculation to decide your sleeping state. While charging it is assumed that you are not wearing the watch and if the status changes to _deep sleep_ the internal heartrate sensor is used to detect if you are wearing the watch.
 
+#### Explanations
+* __Detection of Sleep__  
+  The movement value of bangle's build in health event that is triggered every 10 minutes is checked against the thresholds for light and deep sleep. If the measured movement is lower or eaqual to the __Deep Sleep__-threshold a deep sleep phase is detected for the last 10 minutes. If the threshold is exceeded but not the __Light Sleep__-threshold than the last timeperiod is detected as light sleep phase. On exceeding even this threshold it is assumed that you were awake.
```
Comment (thyttan) [inline apps/sleeplog/README.md:11]: typo: eaqual -> equal

File: apps/sleeplog/README.md [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
+* __True Sleep__  
+  The true sleep value is a simple addition of all registert sleeping periods.
```
Comment (thyttan) [inline apps/sleeplog/README.md:13]: typo: registert -> registered

File: apps/sleeplog/README.md [hunk 3]
```diff
@@ -130,40 +154,65 @@ Available through the App Loader when your watch is connected.
...
+
 
 ---
 ### Worth Mentioning
 ---
 #### To do list
 - Check translations.
 - Add more functionallities to interface.html.
-- Enable recieving data on the Gadgetbridge side + testing.  
+- Enable recieving data on the Gadgetbridge side + testing.
```
Comment (thyttan) [inline apps/sleeplog/README.md:208]: Typo: recieving -> receiving

**Review by thyttan — COMMENTED** [review]
Thanks for the updated readme! Found some typos.

**Comment by thyttan** [comment]
>(After pushing the last commit, I didn't think that was necessary... Would it be enough to just click resolve conversation?)

I'm not entirely sure what your asking and I think I probably wouldn't know the answer even if I did 😅

For me the review comments above your latest commit is tagged 'Outdated' and says they are resolved. 👍

**Comment by gfwilliams** [comment]
Great - thanks!

**Pull request merged**